GestureHandler.js

```diff
@@ -409,6 +410,28 @@ const FlingGestureHandler = createHandler(
   {}
 );
 
+const ForceTouchFallback = props => (
+  <Text style={{ color: 'red' }}>
+    Force Touch is not available on this platform
```

Comment:
I don't think it is a good idea to display text like that. Some people may still only test their stuff on the newest iPhone and never see that until it is too late. Lets keep it render the same way as it used to but add `console.warn` to `componentDidMount` maybe?